boards/arm/adafruit_feather_m0_basic_proto/adafruit_feather_m0_basic_proto.yaml

```diff
@@ -3,7 +3,7 @@ name: Adafruit Feather M0 Basic Proto
 type: mcu
 arch: arm
 ram: 32
-flash: 256
+flash: 232
```

Comment:
This belongs in a separate commit.

Reply:
Or even a separate PR.

Reply:
Separate PR https://github.com/zephyrproject-rtos/zephyr/pull/51618 opened for this change.